handle_tree.go

```diff
@@ -102,19 +102,19 @@ func getTree(
 	}
 
 	list, err := golist(pkg)
-	if err != nil {
+	if err != nil || len(list) != 1 {
```

Comment:
that's quite strange condition, can you explain the second part of it? `len(list) != 1` but if err will be nil, program will exit with empty fatal message

Reply:
I've removed the len check.  It's "C" paranoia creeping in -- since pkg is a string as opposed to []string ==> only one list returned.  I really like Go, but I've only written a little a few years back.